### DISCUSSION TITLE
Looking for other maintainers.

### DISCUSSION THREAD

**Discussion by hallelujah** [hallelujah/valid_email#115] [General]
## You are wanted!!

Due to increase of interests in this gem recently I am seeking 1 or 2 volunteers to maintain this project.

- I (@hallelujah) have basically no time to maintain this. But I would like this gem to continue.
- I prefer people who were active already (PR, issue reports) to apply.
- But I am open to others if they show strong will to help the community

Super power you will get:

- ability to push to rubygems.org
- ability to manage this repository

**Comment by torrocus** [comment]
I can help you maintain this gem if the offer still stands. I probably have more space for it now than you do. Let me know you're alive, please.
